Add some documentation

chalasr opened the pull request.

chalasr committed:
Add some docs

ogizanagi commented on README.md:
```diff
@@ -3,65 +3,175 @@ Manalize
+
+- Work on any new/existing project from any platform in minuts
+- Keep a local environment consistent across projects (practices, tools)
+- Enable/disable support for languages, packages or any various utility as well
```
```diff
- various utility 
+ various utilities
````
?

ogizanagi commented on README.md:
```diff
@@ -3,65 +3,175 @@ Manalize
+- [Composer Semver](https://github.com/composer/semver)
+- [Nikic iter](https://github.com/nikic/iter)
+- [Guzzle](https://github.com/guzzle/guzzle)
+- The Symfony [Console](https://github.com/symfony/console), [Process](https://github.com/symfony/process), [Filesystem](https://github.com/symfony/filesystem), [Yaml](https://github.com/symfony/yaml) and [Stopwatch](https://github.com/symfony/stopwatch) components
```
As discussed, I doubt the `Stopwatch` worth the mention (we may remove it).

ogizanagi commented on README.md:
```diff
@@ -3,65 +3,175 @@ Manalize
+_Note:_
+  
+Be careful when applying the patch, any custom change made to your environment configuration will be erased.  
+To minimize risks, we recommend you to look at the patch before trying to apply it. 
```
See http://nuclearsquid.com/writings/git-add/

maybe we can explain how to proceed (and mention PHPStorm VCS > Apply Patch which allows to apply it interactively)

chalasr replied:
👍

ogizanagi reviewed (approved):
High quality README.md file!! 👏  🍻

chalasr committed:
Fix typos

ogizanagi commented on README.md:
```diff
@@ -14,9 +14,9 @@ Why?
 In short, we need to:
 
-- Work on any new/existing project from any platform in minuts
+- Be able to work on any new/existing project from any platform in minuts
```
```diff
- minuts
+ minutes
```

ogizanagi commented on README.md:
```diff
@@ -61,33 +61,33 @@ $ chmod a+x /usr/local/bin/manalize
+### Checking that your host meets our requirements
 
-Before using `manalize`, you need to ensure that your host is ready. It can easily be achieved by running the following command:
+Before using `manalize`, you need to ensure that your host is ready. It can be easily done by running the following command:
```
I preferred the previous version :/ (`It can easily be achieved `)

ogizanagi commented on README.md:
```diff
@@ -14,9 +14,9 @@ Why?
 
-- Work on any new/existing project from any platform in minuts
+- Be able to work on any new/existing project from any platform in minuts
+- Enable/disable any package or language support as well
```
I preferred the previous version :/ (`Enable/disable support for languages, packages or any various utility as well `)

ogizanagi commented on README.md:
```diff
@@ -61,33 +61,33 @@ $ chmod a+x /usr/local/bin/manalize
 ### Setting up your environment
 
-Given you have a web project that you clone for the first time and you need to run locally, just use the `setup` command:
+Given you have a web project that you clone for the first time and you need to run locally, just execute the `setup` command:
```
```diff
- just execute 
+ simply execute
```

?

ogizanagi commented on README.md:
```diff
@@ -160,7 +160,7 @@ So you can then apply a patch provided by the [diff](#diff) command.
 ---------------
 
-Before all, please [check your requirements](#checking-your-host-requirements) command to ensure that your issue doesn't come from your host.  
+Before all, please [check your host](#checking-your-host-requirements) to ensure that your issue doesn't come from your host.  
```
```diff
- Before all, please [check your host](#checking-your-host-requirements) to ensure that your issue doesn't come from your host.
+ Before all, please [ensure your host satisfies each of our requirements](#checking-your-host-requirements). Your issue(s) may result from unsupported or buggy versions of packages installed on your machine.
```

chalasr commented:
@ogizanagi I addressed all of your comments excepted https://github.com/manala/manalize/pull/87#discussion_r89519960 for which I still need to read the doc and try myself ^^ Thanks a lot for your review

edit: we should address https://github.com/manala/manalize/pull/87#discussion_r89519960
